packages/ui-components/src/index.ts

```diff
@@ -7,3 +7,4 @@ export * from "./Icon/Icon";
 export * from "./FuzzyTime/FuzzyTime";
 export * from "./InlineAlert/InlineAlert";
 export * from "./Spinner/Spinner";
+export * from "./Type/";
```

Comment:
It took me a while to find this components under `Type` dir 🙂 . What do you think about renaming it to `Typography` or something more specific, because it sounds too generic.

Reply:
I like that suggestion. "Type" is a pretty overloaded term in a _TypeScript_ codebase. 😄